ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -187,6 +203,12 @@ module.exports = Marionette.LayoutView.extend({
       },
       error: _this.cleanup(),
     })
+      .done((data, textStatus, jqxhr) => {
+        this.message('Success!', 'Result form successfully saved', 'success')
```

Comment:
❓ We avoid using exclamation points in log messages. Is that also true of messages on the client?

Reply:
@willwill96 I'm not sure, I just wanted to keep this consistent with `search-form-editor.view.js`

Reply:
@andrewkfiedler Should both of the announcements contain `!` or should they both be removed?

Reply:
I would leave it off, but we don't really have any formal guidance on copy yet for the client side.  

@TeresaHudson do you have any strong feelings in this area?